File: VB/App_Code/BootstrapGridViewDataItem.vb
''' <summary>
''' Summary description for BootstrapGridViewDataItem
''' </summary>
Public Class BootstrapGridViewDataItem
    Public Sub New(ByVal id As Integer, ByVal name As String, ByVal type As String)
        Me.ID = id
        Me.Name = name
        Me.Type = type
    End Sub

    Public Property ID() As Integer

    Public Property Name() As String

    Public Property Type() As String
End Class

File: CS/App_Code/BootstrapGridViewDataItem.cs
/// <summary>
/// Summary description for BootstrapGridViewDataItem
/// </summary>
public class BootstrapGridViewDataItem
{
    public BootstrapGridViewDataItem(int id, string name, string type)
    {
        ID = id;
        Name = name;
        Type = type;
    }

    public int ID { get; set; }

    public string Name { get; set; }

    public string Type { get; set; }
}